modules/virt-deprecated-tasks.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-deprecated-tasks_{context}"]
+= Removing deprecated or unused resources
+
+* If you no longer use Tekton (TTO) in your environment after upgrading to OpenShift Container Platform 4.14 or later, you can remove any remaining Tekton-related resources that were previously managed by the ssp-operator.
+
+.Procedure
+
+. Use the following command to remove any leftover Tekton-related resources from the cluster:
```

Comment:
Recommend:
The following command removes any remaining Tekton-related resources from the cluster: